scripts/benchmarks/benchmark_reflex_size.py

```diff
@@ -0,0 +1,204 @@
+"""Checks the size of a specific directory and uploads result."""
+import argparse
+import os
+import subprocess
+from datetime import datetime
+
+import psycopg2
+
+
+def get_directory_size(directory):
```

Comment:
I think this is accurate, just tested this against a few dirs locally. The discrepancies in the db come from the CI, perhaps some packages are added by github. Will need to check that out

Reply:
fixed